src/Harald.WebApi/Application/EventHandlers/SlackAWSContextAccountCreatedEventHandler.cs

```diff
@@ -24,9 +24,14 @@ public async Task HandleAsync(AWSContextAccountCreatedDomainEvent domainEvent)
                 $"Get-ADUser \"CN=IT BuildSource DevEx,OU=DFDS AS,OU=Mailboxes,OU=Accounts,OU=DFDS,DC=dk,DC=dfds,DC=root\" | Set-ADUser -Add @{{proxyAddresses=\"smtp:{domainEvent.Payload.RoleEmail}\"}}";
             var installToolsCmd =
                 $"Get-WindowsCapability -Online | ? {{$_.Name -like 'Rsat.ActiveDirectory.DS-LDS.Tools*'}} | Add-WindowsCapability -Online";
-            var addDeployCredentialsBash = $"ROOT_ID={domainEvent.Payload.CapabilityRootId} ACCOUNT_ID={domainEvent.Payload.AccountId} ./kube-config-generator.sh";
-            var addDeployCredentialsPS = $"./kube-config-generator.ps1 -RootId {domainEvent.Payload.CapabilityRootId} -AccountId {domainEvent.Payload.AccountId}";
-
+            var addDeployCredentialsBash = $"AWS_ROLE_CLOUD_ADMIN=\"[Cloud Administrator Role ARN]\"\\\n" +
+                                           $"AWS_ROLE_ADFS_ADMIN=\"[ADFS Administrator Role ARN]\"\\\n" +
+                                           $"poetry run python ./kube_config_generator.py -r {domainEvent.Payload.CapabilityRootId} -a {domainEvent.Payload.AccountId}";
```

Comment:
There are separate variables for the Bash and PowerShell commands respectively. I suggest creating a separate variable for the Python command too, instead of merging it with the Bash commands.

Reply:
I don't understand the point in us choosing a default scripting language of Python if we then plan to also keep the old bash and powershell versions of the scripts.  It just adds more confusion in my view.

We now have a Python script that does the required task but the way it is executed will vary slightly because the way environment variables are declared.  For this reason I had to update the addDeployCredentialsBash and addDeployCredentialsPS variables to include appropriate declaration methods.

All of the changes i've made are on the basis that once merged in i'll remove the old .ps1 and .sh scripts from the repo so we only use Python going forward.  The idea being that doing so removes to maintain the scripts in the future if anything changes.